### PR TITLE
Quiz app in JS

### PR DESCRIPTION
**Pull request by Aarti002**
issue #321 solved

This app  contains several questions and will continue until user stops it by clicking on "stop" button also questions would be shuffled everytime.

### CODE REVIEW

**Comment by 991rajat** [comment]
Hey @Aarti002 Sorry for the delay, there is miscommunication currently there are quiz based project in our current codebase.
What I would suggest you should change the questions to something like puzzle questions simple short puzzle questions and there answers, this way it will be get merged.
Change the title of app to Puzzle Quiz App.
Also add the project to script.js
Sorry for last minute changes

**Comment by Aarti002** [comment]
so shall i change the questions to puzzle type question something which involve tricks or something, right? @991rajat 
Thank you!

**Comment by 991rajat** [comment]
Yes Just change the question to something else, What ever you like, 
which are not present in our current code
Math Quiz and JS quiz is present in our current code.
So pick any subject you like and replace the questions with them

**Comment by Aarti002** [comment]
yes i'm already working on it!
Thank you

**Comment by Aarti002** [comment]
hey @991rajat please review and merge my project , if something is missing let me know!
Thank you!!

**Comment by 991rajat** [comment]
@Aarti002 Please add the project to script.js.

**Comment by Aarti002** [comment]
okay will do it

**Comment by Aarti002** [comment]
Can you see my project now @991rajat ?

File: script.js [hunk 1]
```diff
@@ -349,6 +349,11 @@ let projectData = [
     projectImage: "assets/GIFs/wordScramble.gif",
     projectUrl: "public/wordScramble.html",
   },
+  {
+    projectName: "Puzzle quiz",
+    projectImage: "assets/Images/quiz",
```
Comment (991rajat) [inline script.js:354]: Add a proper image. It;s not showing
image

**Review by 991rajat — COMMENTED** [review]
Looks Good, just fix the image issue.

**Comment by Aarti002** [comment]
@991rajat i think address of image folder of my project is correct or maybe i'm confuse don't know please help.
Thank you!

**Comment by 991rajat** [comment]
Path is not correct, as there is no image there. You need to add the path of image preview.
If you have the image just fix the path.
If don't
Create a image preview of the project.
And save it in assets/Images/quiz/quizPicture.jpg
and fix the path.

**Comment by Aarti002** [comment]
@991rajat Can you check again this time it might work.
thanks!

**Comment by 991rajat** [comment]
Hey Now images inside the project is not viewable.
Before it was working Now its not you can go back whatever previous changes were.
image
What I told you to fix is on main project your project is visible but there is no image as others have
image
what you need to do is to create a image of your project and add the path to script.js so that in projects it is visible

**Comment by Aarti002** [comment]
@991rajat Now it may work fine : )
Thanks for your reviews!

**Comment by 991rajat** [comment]
Looks good. One thing more don't push setting.json 
Fix that please.

**Comment by Aarti002** [comment]
but i haven't done anything with that file @991rajat , i m confuse actually can you explain?

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (991rajat) [inline .vscode/settings.json:2]: This file.

**Review by 991rajat — COMMENTED** [review]
You don't have to push this.

**Comment by 991rajat** [comment]
Just delete this file and push again.

**Comment by Aarti002** [comment]
Ohhk , got it thank you so much @991rajat

**Comment by 991rajat** [comment]
Delete the file, please and folder too.
You just remove the contents of it.

**Comment by 991rajat** [comment]
Wait

**Comment by 991rajat** [comment]
No problems I could fix from the UI.

**Review by 991rajat — APPROVED** [review]
Good Work.

**Comment by Aarti002** [comment]
is everything okay now? @991rajat

**Comment by Aarti002** [comment]
Thank you so much!!

**Comment by 991rajat** [comment]
Thank you for this awesome contribution. 🤩 
Keep Contributing.

**Pull request merged**